.circleci/config.yml

```diff
@@ -18,7 +18,18 @@ jobs:
           name: 'pylint'
           command: |
             source /usr/local/share/virtualenvs/tap-dynamodb/bin/activate
-            make lint
+            pylint tap_dynamodb --disable 'duplicate-code,missing-module-docstring,missing-class-docstring,too-few-public-methods,line-too-long,fixme,missing-function-docstring,raise-missing-from,consider-using-f-string,too-many-locals,invalid-name,too-many-arguments'
```

Comment:
@namrata270998 Do we need this change for any specific reason?.

Reply:
reverted the change